tutorretirement/templates/retirement/apps/retirement/run_retirement_pipeline.sh

```diff
@@ -0,0 +1,14 @@
+cool_off_days=$1
+pip install --upgrade pip && pip install -r scripts/user_retirement/requirements/base.txt --exists-action w
```

Comment:
Just so I understand this correctly: retirement has moved into the platform repo, but the packages needed for retirement don't actually get baked into the default Tutor `openedx` image, meaning we have to install them every time we run the retirement scripts?

Reply:
Yes, those requirements are under the `/scripts/user_retirement` directory and not installed in the `openedx` image.

Reply:
OK, doesn't that run counter to the idea of having pre-built container images though?

(In other words: should we perhaps continue to build images, just as we did prior to Redwood?)

Reply:
Hmm, okay, I guess we can do that also yes.

Reply:
Should I go ahead and update this with building the image?

Reply:
I think so, yes.